backpack-lint/src/main/java/net/skyscanner/backpack/lint/check/BpkComposeComponentUsageDetector.kt

```diff
@@ -98,6 +98,11 @@ class BpkComposeComponentUsageDetector : Detector(), SourceCodeScanner {
       webName = "dialog",
       componentsToReplace = setOf("androidx.compose.ui.window.Dialog", "androidx.compose.material.AlertDialog")
     ),
+    FAB(
+      fullName = "net.skyscanner.backpack.compose.fab.BpkFab",
+      webName = "floating-action-button",
+      componentsToReplace = setOf("androidx.compose.material.FloatingActionButton")
```

Comment:
Let's also add another component to this rule – ExtendedFloatingActionButton located in the same package.